web-registry/src/features/ecocredit/CreateBatchBySteps/CreateBatchMultiStepForm/CreditBasics.tsx

```diff
@@ -1,39 +1,39 @@
-import React, { useState, useEffect } from 'react';
-import { useFormikContext, Field, FieldArray } from 'formik';
-import * as Yup from 'yup';
-import { isPast } from 'date-fns';
+import React, { useEffect, useState } from 'react';
 import { Box, IconButton, Link } from '@mui/material';
-import { useTheme, DefaultTheme as Theme } from '@mui/styles';
+import { DefaultTheme as Theme, useTheme } from '@mui/styles';
+import { isPast } from 'date-fns';
+import { Field, FieldArray, useFormikContext } from 'formik';
+import * as Yup from 'yup';
 
-import { Body } from 'web-components/lib/components/typography';
-import InputLabel from 'web-components/lib/components/inputs/InputLabel';
-import OnBoardingCard from 'web-components/lib/components/cards/OnBoardingCard';
-import TextField from 'web-components/lib/components/inputs/TextField';
-import { DatePickField } from 'web-components/lib/components/inputs/DatePickField';
 import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+import OnBoardingCard from 'web-components/lib/components/cards/OnBoardingCard';
 import TrashIcon from 'web-components/lib/components/icons/TrashIcon';
-import { AddCertificationModal } from 'web-components/lib/components/modal/AddCertificationModal';
+import { DatePickField } from 'web-components/lib/components/inputs/DatePickField';
+import InputLabel from 'web-components/lib/components/inputs/InputLabel';
+import SelectTextField, {
+  Option,
+} from 'web-components/lib/components/inputs/SelectTextField';
+import TextField from 'web-components/lib/components/inputs/TextField';
 import {
-  requiredMessage,
   invalidDate,
+  invalidJSON,
   invalidPastDate,
-  vcsRetirementSerialRE,
   invalidVCSRetirement,
   isValidJSON,
-  invalidJSON,
+  requiredMessage,
+  vcsRetirementSerialRE,
 } from 'web-components/lib/components/inputs/validation';
+import { AddCertificationModal } from 'web-components/lib/components/modal/AddCertificationModal';
+import { Body } from 'web-components/lib/components/typography';
 import { NameUrl } from 'web-components/lib/types/rdf';
 import { VCSBatchMetadataLD } from 'web-components/lib/types/rdf/C01-verified-carbon-standard-batch';
-import { Option } from 'web-components/lib/components/inputs/SelectTextField';
-import SelectTextField from 'web-components/lib/components/inputs/SelectTextField';
 
-import useQueryProjectsByIssuer from '../../../../hooks/useQueryProjectsByIssuer';
 import { MetadataJSONField } from '../../../../components/molecules';
+import useQueryProjectsByIssuer from '../../../../hooks/useQueryProjectsByIssuer';
 import { useWallet } from '../../../../lib/wallet';
-
+import useSaveProjectSelectedOption from '../hooks/useSaveProjectSelectedOption';
 import useUpdateProjectClass from '../hooks/useUpdateProjectClass';
 import useUpdateProjectOptions from '../hooks/useUpdateProjectOptions';
-import useSaveProjectSelectedOption from '../hooks/useSaveProjectSelectedOption';
 
```

Comment:
I can't tell what happened here, but this page is not working right. It was working in this [change](https://github.com/regen-network/regen-web/pull/1065/) , but validation and initial form state are not right.